.github/workflows/infra_apply.yaml

```diff
@@ -100,29 +100,14 @@ jobs:
             printf "%q\n" "$i" >> "$GITHUB_ENV"
           done
 
-      - name: Azure Login
-        uses: azure/login@a65d910e8af852a8061c627c456678983e180302 # v2
-        with:
-          client-id: ${{ env.ARM_CLIENT_ID }}
-          tenant-id: ${{ env.ARM_TENANT_ID }}
-          subscription-id: ${{ env.ARM_SUBSCRIPTION_ID }}
+      - uses: ./.github/actions/azure-login
+        name: Azure Login
 
-      - name: Set Terraform Version
+      - name: Terraform Setup and Init
```

Comment:
In reference to my [previous comment](https://github.com/pagopa/dx/pull/356/files#r1998809670), this name is showing the misunderstanding that may arise from the user